README.md

```diff
@@ -51,9 +33,22 @@ roseus hrp2jsknt-interface.l
 
 ### Package Description
 
-hrpsys_ros_bridge_tutorialsls
+### [hrpsys_ros_bridge_tutorials]
 --------------
-This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common). 
+This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common).
 
 Euslisp tutorial wiki is written in [ROS wiki](http://wiki.ros.org/rtmros_common/Tutorials/WorkingWithEusLisp).
 
+- You should prepare robot model file. Supported types of model file are collada(openrave) and VRML(openhrp3). URDF and OpenRAVE xml can be used by converting to collada.
```

Comment:
Maybe we cannot get what we should actually do from `You should prepare robot model file`.
How about noting actual operation (for example, `You should prepare robot model file (e.g., no additional operation is required for SampleRobot, compiling hrp2_models is required for HRP2)`)

Reply:
Thank you. I fixed it.